policy/approval/approve.go

```diff
@@ -207,7 +207,7 @@ func (r *Rule) IsApproved(ctx context.Context, prctx pull.Context) (bool, string
 		return false, msg, nil
 	}
 
-	msg := fmt.Sprintf("%s needed", numberOfApprovals(remaining))
+	msg := fmt.Sprintf("%s needed (of required %d)", numberOfApprovals(remaining), r.Requires.Count)
```

Comment:
What do you think about formatting this as `1/n approvals required`? I supposed we'd have to change how `numberOfApprovals`, but it might be more natural to read that the parenthetical suffix.

Related, should we update the message on L204 as well?